Makefile

```diff
@@ -844,3 +844,12 @@ run_all_use_case_examples:
 .PHONY: check_utils_use_case # Check that no utils.py are found in use_case_examples
 check_utils_use_case:
 	./script/make_utils/check_utils_in_use_case.sh
+
+.PHONY: check_symlinks # Check that no utils.py are found in use_case_examples
+check_symlinks:
```

Comment:
needs to be added in the CI or as a sub target of something we already launch, no?

Reply:
could be done in the `- name: Check links` step